pkg/collector/corechecks/embed/process_agent.go

```diff
@@ -129,7 +129,7 @@ func (c *ProcessAgentCheck) run() error {
 func (c *ProcessAgentCheck) Configure(data integration.Data, initConfig integration.Data, source string) error {
 	// handle the case when the agent is disabled via the old `datadog.conf` file
 	if enabled := config.Datadog.GetBool("process_config.enabled"); !enabled {
-		return fmt.Errorf("Process Agent disabled through main configuration file")
+		log.Info("Process Agent disabled through main configuration file")
```

Comment:
Shouldn't we return nil here ?

Reply:
+1

Reply:
I think this function is only meant to parse all the configs. The idea is to always let process-agent run and use the config to stop process-agent once it's started. If we break early then there are several things that didn't get initialized(like binPath) so when process-agent gets running there'll be one or two unnecessary(non critical) errors(like fork error).

Reply:
If `binPath` is not initialized you'll see error:
```
2019-09-03 11:35:50 EDT | CORE | ERROR | (pkg/collector/runner/runner.go:292 in work) | Error running check Process Agent: fork/exec : no such file or directory
```

Reply:
Merging to build new RC, but would be nice to add this explanation as a comment in the code, as jaime suggested (on a new PR).